ovirt-ansible-disaster-recovery.spec.in

```diff
@@ -37,6 +37,11 @@ sh build.sh install
 %license LICENSE
 
 %changelog
+* Thu Jul 19 2018 Maor Lipchuk <mlipchuk@redhat.com> - 1.1-2
+- Bug 1576934 - Support dc name for vnic profile mapping
```

Comment:
Also it's common to use `rhbz#1576934` instead of `Bug xxx`

Reply:
done